flowpipe/node.py

```diff
@@ -304,29 +313,25 @@ def node_repr(self):
                 pretty += '-->'
             else:
                 pretty += offset
-
-            value = ""
-            if in_plug.value is not None:
-                value = json.dumps(in_plug.value, cls=NodeEncoder)
             plug = '{symbol} {dist}{input_}{value}'.format(
                 symbol='%' if in_plug._sub_plugs else 'o',
                 dist=' ' if isinstance(in_plug, SubInputPlug)else '',
                 input_=input_,
-                value=('<{value}>'.format(value=''.join(
-                    [s for i, s in enumerate(str(value))
-                     if i < max_value_length]))
-                    if not in_plug._sub_plugs else ''))
+                value=_short_value(in_plug))
             pretty += '{plug:{width}}|'.format(plug=plug, width=width + 1)
 
         # Outputs
         all_outputs = self.all_outputs()
```

Comment:
Is this duplicating l. 267? If so, I'd suggest removing l. 267 and keeping l. 324.

Reply:
Right, thanks @neuneck